src/data/formatters.js

```diff
@@ -25,13 +26,12 @@ export const formatPrice = (price: number) => {
   return price.toFixed(2);
 };
 
-export const formattedEventPrice = (
-  isFree: boolean,
+export const formatShortEventPrice = (
   eventPriceLow: number,
   eventPriceHigh: number
 ) => {
   let displayPrice;
-  if (isFree) {
+  if (isFree(eventPriceLow, eventPriceHigh)) {
```

Comment:
Not going to block, but you can refactor function to return early for each case instead of using a `let` statement.